### PR TITLE
Updated BIGTOP-2467.patch

### PR DESCRIPTION
**Pull request by petevg**
Got merged upstream, so we could get rid of it in the "master"
directory. Updated the patch in the 1.1 directory, so that it contains
my backwards compatible changes.

@juju-solutions/bigdata

### CODE REVIEW

File: resources/bigtop-1.1.0/BIGTOP-2467.patch [hunk 1]
```diff
@@ -43,6 +43,71 @@ index 69d98c4..b646e60 100644
...
+-<% @ensemble.each do |idx, ip| %>
++<% if !@ensemble[0].nil? && @ensemble[0].length == 2 %>
++  <% @ensemble.each do |idx, ip| %>
+ server.<%= idx %>=<%= ip %>
++  <% end %>
++<% else %>
++  <% @ensemble.each_with_index do |server,idx| %>
++server.<%= idx %>=<%= server %>
++  <% end %>
+ <% end %>
```
Comment (petevg) [inline resources/bigtop-1.1.0/BIGTOP-2467.patch]: I had github generate the patch this time, so it's all slightly different. Line 103-110 are the only differences from the main patch, though.

**Comment by petevg** [comment]
Zookeeper tests pass, with the updated patch, running with the latest juju beta against LXD containers.

**Comment by johnsca** [comment]
I think the large diff is due to the difference between the squashed merge vs the multiple commits.  Perhaps we could use https://github.com/apache/bigtop/commit/87e7d01da2558347f9ea64983506d9d8d944a4b2.patch as the source instead, as that's what actually got in upstream?

Also, https://github.com/apache/bigtop/pull/116 should probably be closed.

**Comment by johnsca** [comment]
:+1:

**Pull request merged**